builtin/logical/ssh/path_roles.go

```diff
@@ -374,6 +375,18 @@ func pathRoles(b *backend) *framework.Path {
 					Value: 30,
 				},
 			},
+
+			"not_before_absolute": {
+				Type: framework.TypeTime,
```

Comment:
Interesting, I had not used `TypeTime` before; we had went with `TypeString` for PKI.

I think this should be compatible both ways, but I don't think you'll [ever get a string as output](https://github.com/openbao/openbao/blob/1044cd544c9f5011c16510fe13d15061132224ed/sdk/framework/field_data.go#L268-L279) below. :-)